recipe/meta.yaml

```diff
@@ -22,17 +21,19 @@ requirements:
     - pip
     - pytest
     - python
+    - setuptools
   run:
     - amqp >=5.1.1,<6.0.0
-    - backports.zoneinfo >=0.2.1    # [py<39 or python_impl == "pypy"]
-    - python-tzdata  # [py<39 or python_impl == "pypy"]
+    - backports.zoneinfo >=0.2.1    # [py<39]
+    - python-tzdata  # [py<39]
     - typing_extensions  # [py<310]
```

Comment:
It's honestly ok to always include this. This kind of micro optimization is not strictly necessary.

Reply:
I thought so